kmip/services/server/session.py

```diff
@@ -127,7 +127,8 @@ def _handle_message_loop(self):
         )
 
         try:
-            if hasattr(self._connection, 'shared_ciphers') and self._connection.shared_ciphers() is not None:
+            if hasattr(self._connection, 'shared_ciphers') \
+                    and self._connection.shared_ciphers() is not None:
```

Comment:
Parentheses are usually preferable to escaped newlines.

```suggestion
            if (hasattr(self._connection, 'shared_ciphers')
                    and self._connection.shared_ciphers() is not None):
```